metricbeat/docs/modules/kubernetes.asciidoc

```diff
@@ -79,6 +79,7 @@ metricbeat.modules:
   add_metadata: true
   # When used outside the cluster:
   #host: node_name
+  # It defaults to `KUBECONFIG` environment variable if present.
```

Comment:
From my POV this is not accurate.
A user could think that even if the `kube_config` element is informed, `KUBECONFIG` will overwrite it, but that's not the case.

Something like `if kube_config is not informed, KUBECONFIG env var will be checked and if not present it will fall back to InCluster`

WDYT?